CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [2.1.3](https://github.com/Workiva/r_tree/compare/2.1.2...2.1.3)
```

Comment:
Might as well add 2.1.4 since it will release when this is merged.
```suggestion
## [2.1.4](https://github.com/Workiva/r_tree/compare/2.1.3...2.1.4)
```